libs/application/templates/driving-license/src/fields/AdvancedLicense/AdvancedLicense.tsx

```
@@ -0,0 +1,146 @@
+
+        const s2 = options.find((x) => x.minAge)?.minAge
+
+        const requiredAgeText =
+          s1 &&
+          s2 &&
+          formatMessage(m[`applicationForAdvancedAgeRequired`])
+            ?.replace('%1', s1)
+            ?.replace('%2', String(s2))
+
+        return (
+          <Box marginTop={index === 0 ? 2 : 5} marginBottom={5}>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)